pkg/apis/networking/types.go

```diff
@@ -339,15 +339,12 @@ type IngressClassParametersReference struct {
 	Name string
 	// Scope represents if this refers to a cluster or namespace scoped resource.
 	// This may be set to "Cluster" (default) or "Namespace".
-	// Field can be enabled with IngressClassNamespacedParams feature gate.
 	// +optional
-	// +featureGate=IngressClassNamespacedParams
 	Scope *string
 	// Namespace is the namespace of the resource being referenced. This field is
 	// required when scope is set to "Namespace" and must be unset when scope is set to
 	// "Cluster".
 	// +optional
-	// +featureGate=IngressClassNamespacedParams
```

Comment:
Also need to change the v1 and v1beta1 types:

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/networking/v1beta1/types.go#L338-L349
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/networking/v1/types.go#L535-L546

Reply:
Thanks for catching this. Updated.